assets/src/application/ui/layout/entityList/filterBar/buttons/ToggleLegendButton.tsx

```diff
@@ -19,7 +20,7 @@ const ToggleLegendButton: React.FC<ToggleLegendButtonProps> = ({ listId, showLeg
 		</animated.div>
 	);
 	const filterId = `ee-toggle-legend-btn-${listId}`;
-	const tooltip = __(`${showLegend ? 'hide' : 'show'} legend`);
+	const tooltip = showLegend ? __('hide legend') : __('show legend');
```

Comment:
@nicolad please note that translation functions cannot handle variables. If you want use variables, then use `sprintf`.

Reply:
ok, thx for explanation